systems/notification/mailer/pkg/config.go

```diff
@@ -9,33 +9,24 @@
 package pkg
 
 import (
-	"fmt"
-	"os"
-	"strconv"
-	"strings"
 	"time"
 
-	"github.com/sirupsen/logrus"
 	uconf "github.com/ukama/ukama/systems/common/config"
 )
 
-type MailerConfig struct {
-	Host     string `default:"localhost"`
-	Port     int    `default:"587"`
-	Username string `default:""`
-	Password string `default:""`
-	From     string `default:"hello@dev.ukama.com"`
-}
-
 type Config struct {
 	uconf.BaseConfig `mapstructure:",squash"`
 	DB               *uconf.Database `default:"{}"`
 	Grpc             *uconf.Grpc     `default:"{}"`
 	Queue            *uconf.Queue    `default:"{}"`
 	Timeout          time.Duration   `default:"50s"`
 	Service          *uconf.Service
-	Mailer           *MailerConfig
 	TemplatesPath    string `default:"templates"`
+	MailerHost       string `default:"localhost"`
+	MailerPort       int    `default:"587"`
+	MailerUsername   string `default:""`
+	MailerPassword   string `default:""`
+	MailerFrom       string `default:"hello@dev.ukama.com"`
```

Comment:
Previous implementation was fine.
```
type MailerConfig struct {
	Host     string `default:"localhost"`
	Port     int    `default:"587"`
	Username string `default:""`
	Password string `default:""`
	From     string `default:"hello@dev.ukama.com"`
}
```
& 
`Mailer           *MailerConfig`

What's the reason of change?